### PR TITLE
created markdown text area

### LINKED ISSUE #26: Create a Markdown Text Area
We should create a markdown text area to our site.

We should also add a parser to it.

### PR DESCRIPTION
**Pull request by Maher9596**
I created a markdown text area I used **showdown** package for parsing.

This PR is to close issue #26

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 dist/
 .cache/
+node_modules/
```
Comment (Ma7eer) [inline .gitignore]:
```suggestion
node_modules/

```

File: mdt.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+const showdown = require('showdown')
+
+const converter = new showdown.Converter()
+
+textEditor.addEventListener("keyup", event => {
+    const { value } = event.target
+
+    const html = converter.makeHtml(value)
+
+    preview.innerHTML = html;
```
Comment (Ma7eer) [inline mdt.js]:
```suggestion
    preview.innerHTML = html
```

**Review by Ma7eer — CHANGES_REQUESTED** [review]
Minor items to change

**Comment by Maher9596** [comment]
I committed the changes.

**Review by Ma7eer — APPROVED** [review]

**Pull request merged**